common/sampling.cpp

```diff
@@ -103,96 +101,89 @@ std::string llama_sampling_print(const llama_sampling_params & params) {
 
 std::string llama_sampling_order_print(const llama_sampling_params & params) {
     std::string result = "CFG -> Penalties ";
-    
-    std::unordered_map<char, std::string> samplers_map_display {
-        {'k',     "-> top_k "},
-        {'f',     "-> tfs_z "},
-        {'y', "-> typical_p "},
-        {'p',     "-> top_p "},
-        {'m',     "-> min_p "},
-        {'t',      "-> temp "}
-    };
```

Comment:
I think using a map for the labels is fine. Either that or use one line per case in the switch, but perhaps ggerganov wouldn't like that code style.

Reply:
You're right, there are some switches in llama.cpp with one-line cases.